docs/sources/permissions/dashboard_folder_permissions.md

```diff
@@ -71,3 +71,22 @@ Result: You cannot override to a lower permission. `user1` has Admin permission
 For example if "Everyone with Editor Role Can Edit" exists in the ACL list then **John Doe** will still have Edit permission even after you have specifically added a permission for this user with the permission set to **View**. You need to remove or lower the permission level of the more general rule.
 - You cannot override permissions for users with **Org Admin Role**
 - A more specific permission with lower permission level will not have any effect if a more general rule exists with higher permission level. For example if "Everyone with Editor Role Can Edit" exists in the ACL list then **John Doe** will still have Edit permission even after you have specifically added a permission for this user with the permission set to **View**. You need to remove or lower the permission level of the more general rule.
+
```

Comment:
I recommend making the tasks into separate topics. The title is fine, but you would want to add a one-sentence or so intro to each. Something like:

Grant folder permissions to a user or a team.
Grant dashboard permissions to a user or a team.